extensions/shared/jetpack-likes-and-sharing-panel.js

```diff
@@ -12,7 +14,15 @@ JetpackLikesAndSharingPanel.Slot = () => (
 	<Slot>
 		{ fills => {
 			if ( ! fills.length ) {
-				return null;
+				const sharingAdminLink = '/wp-admin/admin.php?page=jetpack#/sharing';
+				// const sharingAdminLink = addQueryArgs( "/wp-admin/admin.php", { page: "jetpack#/sharing" } );
+				// const sharingAdminLink = getWPAdminURL( 'admin.php', { page: 'jetpack#/sharing' } );
+				return (
+					<PanelBody title={ __( 'Likes and Sharing', 'jetpack' ) }>
+						Visit <a href={ sharingAdminLink }>Sharing settings</a> in your Dashboard to allow
+						visitors to like and share your posts.
```

Comment:
Note that Publicize has a similar CTA:

<img width="284" alt="image" src="https://user-images.githubusercontent.com/87168/56113264-06638880-5f66-11e9-928d-862049781afc.png">

These both CTAs might need small adjustments to ensure customers know which features they're enabling from the two CTAs.